packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -69,6 +72,48 @@ export interface AliasConfiguration {
   readonly securityPolicy?: SecurityPolicyProtocol;
 }
 
+/**
+ * Configuration for custom domain names
+ *
+ * CloudFront can use a custom domain that you provide instead of a
+ * "cloudfront.net" domain. To use this feature you must provide the list of
+ * additional domains, and the ACM Certificate that CloudFront should use for
+ * these additional domains.
+ */
+export interface AliasConfig {
+  /**
+   * AWS Certificate Manager (ACM) certificate.
+   */
+  readonly acmCert: certificatemanager.ICertificate;
```

Comment:
No abbreviations. `acmCertificate`. I think we might even just do `certificate`, no?